src/Microsoft.IdentityModel.Tokens/AsymmetricSecurityKey.cs

```diff
@@ -27,8 +27,14 @@
 
 namespace Microsoft.IdentityModel.Tokens
 {
+    /// <summary>
+    /// base class for Asymmetric Security Key.
+    /// </summary>
     public abstract class AsymmetricSecurityKey : SecurityKey
     {
+        /// <summary>
+        /// true if there is a private key; otherwise, false.
```

Comment:
Because this is abstract, I would expect that the description would contain something like "This can be overridden to specify whether the key has a private key component."